InteractionBase/src/vis/ViewSwitcherMenu.cpp

```diff
@@ -87,7 +87,7 @@ void ViewSwitcherMenu::endFocusMode(Visualization::Item *target)
 		QPoint pos = indexOf(entry);
 		auto nameAfter = entry->nameField()->text();
 		if (scene()->viewItems()->viewItem(nameAfter) == nullptr && nameAfter != nameBefore_)
-			scene()->viewItems()->newViewItem(nameAfter, pos);
+			scene()->viewItems()->createIfNotExists(nameAfter, pos);
```

Comment:
I don't like this name, I guess mainly because of the `Not`. What about:
* `createIfNew()` // not so great to have an `If` in a method name
* `createOrOpen()` // I guess you don't really open anything, but just a thought
* something else that is more straight forward and precise